template/Gemfile

```diff
@@ -13,4 +13,4 @@ gem 'govuk_tech_docs'
 
 # Overrride middleman-search with our fork.
 # See: https://github.com/manastech/middleman-search/pull/24
-gem 'middleman-search', :git => "git://github.com/alphagov/middleman-search.git"
+gem 'middleman-search', git: 'https://github.com/alphagov/middleman-search'
```

Comment:
Perhaps the tests fail because it's missing `.git`?

https://bundler.io/v1.12/git.html